Model/Generator.php

```diff
@@ -631,6 +631,23 @@ private function completeCheckout($comment, RequestInterface $request)
                 if (strtolower($paymentType) === 'paymentandcapture' || strtolower($paymentType) === 'subscriptionandcharge') {
                     $this->paymentHandler->createInvoice($order);
                 }
+
+                // Properly deactivate the quote when order is successful
+                if ($status === 'succeeded' || $status === 'success') {
```

Comment:
Didn't we agree to using the reservation and captured amount to see whether a transaction is successful?

Reply:
@emicha  I’ve removed the condition because the transaction is already validated earlier in the flow. Since this callback only handles successful responses, the additional check was unnecessary.